mubench.reviewsite/templates/includes/review_modal.phtml

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * @var string $project
+ * @var string $version
+ * @var Misuse $misuse
+ * @var array $run all run information
+ * @var array $additional_stats all stat columns from the run information besides project, version, runtime, result and number_of_findings
+ * @var Parsedown $markdown_parser markdown parser
+ */
+
+use MuBench\ReviewSite\Models\Misuse;
+$markdown_parser = new Parsedown();
+?>
+
+<i value="<?= htmlspecialchars($project) ?>_<?= htmlspecialchars($version) ?>_<?= htmlspecialchars($misuse->id) ?>" onclick="showModal(this)" class="fa fa-comments-o clickable-icon" aria-hidden="true"></i>
+<div id="<?= htmlspecialchars($project) ?>_<?= htmlspecialchars($version) ?>_<?= htmlspecialchars($misuse->id) ?>" class="modal">
+    <div class="modal-content">
+                            <span class="close"
+                                  id="<?= htmlspecialchars($project) ?>_<?= htmlspecialchars($version) ?>_<?= htmlspecialchars($misuse->id) ?>">&times;</span>
+        <table class="invisible">
+            <tr><td><b>Detector:</b></td><td><?= htmlspecialchars($detector->muid); ?></td></tr>
+            <tr><td><b>Target:</b></td><td>project '<?= $markdown_parser->text($misuse->getProject()); ?>' version <?= $markdown_parser->text($misuse->getVersion()); ?></td></tr>
+            <tr><td><b>Misuse:</b></td><td>misuse '<?= $markdown_parser->text($misuse->misuse_muid); ?>'</td></tr>
+            <tr><td><b>Tags:</b></td>
+                <td>
+                    <?php foreach($misuse->misuse_tags as $tag) : ?>
+                        <div class="misuse-tag">
+                            <span><?= htmlspecialchars($tag->name) ?></span>
+                        </div>
+                    <?php endforeach; ?>
+                </td>
+            </tr>
+        </table>
+        <h2>Reviews:</h2>
+        <table class="invisible">
+            <tr>
+            <?php foreach($misuse->reviews as $review) : ?>
+            <td>
+                <div class="card-shadow">
```

Comment:
Change this to `div` with `float:left` to make it relayout when the window gets too small.